### PR TITLE
Sample App Update

### PR DESCRIPTION
**Pull request by abhishek-engg-mobilesdk**
- MoEngage SDK Version Update
- Cards Module Integration
- Code Documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,88 @@
...
+|------------------------|--------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Core Plugin            | react-native-moengage          | [![npm](https://img.shields.io/npm/v/react-native-moengage.svg?maxAge=2592000)](https://www.npmjs.com/package/react-native-moengage)                     |
+| Inbox Plugin           | react-native-moengage-inbox    | [![npm](https://img.shields.io/npm/v/react-native-moengage-inbox.svg?maxAge=2592000)](https://www.npmjs.com/package/react-native-moengage-inbox)         |
+| Geofence Plugin        | react-native-moengage-geofence | [![npm](https://img.shields.io/npm/v/react-native-moengage-geofence.svg?maxAge=2592000)](https://www.npmjs.com/package/react-native-moengage-geofence)   |
+| Cards Plugin           | react-native-moengage-cards    | [![npm](https://img.shields.io/npm/v/react-native-moengage-cards.svg?maxAge=2592000)](https://www.npmjs.com/package/react-native-moengage-cards)         |
```
Comment (umangmoe) [inline README.md:15]: fix badge.

File: README.md [hunk 2]
```diff
@@ -1 +1,88 @@
...
+|-------------------------------------------|----------------------|-------------------------------------------------------------------------------------------------------------|
+| MoEngage Android SDK                      | moe-android-sdk      | ![MavenBadge](https://maven-badges.herokuapp.com/maven-central/com.moengage/moe-android-sdk/badge.svg)      |
+| MoEngage Inbox Core                       | inbox-core           | ![MavenBadge](https://maven-badges.herokuapp.com/maven-central/com.moengage/inbox-core/badge.svg)           |
+| MoEngage Goefence Library                 | geofence             | ![MavenBadge](https://maven-badges.herokuapp.com/maven-central/com.moengage/geofence/badge.svg)             |
+| MoEngage Cards Core                       | cards-core           | ![MavenBadge](https://maven-badges.herokuapp.com/maven-central/com.moengage/cards-core/badge.svg)           |
+
+#### iOS
+Install the native MoEngage iOS SDK by using CocoaPods as described here: https://docs.moengage.com/docs/sdk-integration#section-integration-through-cocoapods
```
Comment (umangmoe) [inline README.md:79]: remove the link to docs.moengage.com

Comment (abhishek-engg-mobilesdk) [reply README.md]: removed

File: README.md (hunk 1 above)
Comment (abhishek-engg-mobilesdk) [reply README.md]: updated

**Review by umangmoe — APPROVED** [review]

**Pull request merged**